paket.dependencies

```diff
@@ -44,3 +44,10 @@ group Build
   nuget ILRepack
   
   github fsharp/FAKE modules/Octokit/Octokit.fsx
+  
+  nuget 0x53A.ReferenceAssemblies.Paket 0.2.0
```

Comment:
So every project now bundles their own reference assemblies :P

Reply:
Or put differently, if you think this is a good way forward (which I think it is). We might want to have another documented project on how this stuff works (Then others can use it and read about it). Anyway that's just a suggestion (I'd like to maybe add that to FAKE as well)

Reply:
Yeah, I would prefer to use an official package, also.

The one advantage this has is that it only bundles the directories it actually needs, so the size is a bit smaller. I _could_ have given it a specific name like "ReferenceAssemblies.Net45", but this way it is clear that it is used by paket and will change as soon as we update.

Building the package is dead easy, clone https://github.com/jaredpar/xcopy-msbuild, modify which frameworks to include https://github.com/jaredpar/xcopy-msbuild/blob/006e1bb65d031854f38eaac251b5f799ce281e37/build-reference-assemblies.ps1#L23-L30
and run the powershell script.

Reply:
Maybe we should add that as comment then?